src/main/java/Binks.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Binks {
+    /**
+     * Prints out the greeting message from the chatbot "Binks".
+     */
+    public static void greetUser(){
+        createLineSpacing();
+        System.out.println("Hello! I'm Binks.");
+        System.out.println("What can I do for you?");
+        createLineSpacing();
+    }
+
+    /**
+     * Prints out the line spacing between inputs and outputs in the console.
+     */
+    public static void createLineSpacing(){
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Prints out the exit message from the chatbot "Binks" when user inputs "Bye".mark
+     */
+    public static void exitChatbot(){
+        createLineSpacing();
+        System.out.println("Bye. Hope to see you again soon!");
+        createLineSpacing();
+    }
+    public static void main(String[] arg){
+        greetUser();
+        List list = new List();
+        while(true){
+            String line;
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+            String[] command = line.split(" ");
+            if (line.equalsIgnoreCase("list")){
+                list.getList();
+            }
+            //when the mark command is given
+            else if(command.length == 2 && command[0].equalsIgnoreCase("mark")){
```

Comment:
Perhaps consider refactoring your if and if else boolean statements to make it more readable